lib/gen_stage.ex

```diff
@@ -903,6 +921,12 @@ defmodule GenStage do
     end
   end
 
+  defp noreply_callback(:handle_demand, [demand, state], %{mod: mod, type: :producer_consumer} = stage) do
```

Comment:
On it. But ( excuse the noob question ) a producer_consumer might not be subscribed to a producer here?